js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,10 @@
+function abbreviateString(str) {
+  if (typeof (str) !== 'string') { throw Error(`Expected string, got ${typeof str}`); }
+
+  const stringArray = str.split(' ');
+  const startString = stringArray[0];
```

Comment:
From ReadMe, I assume we meant `firstWord` of String. Because `startString` can be mean different for different people.
what do you think?